Fix/hpa metric issue

kartik-nighania opened the pull request:
Thanks for the amazing Chart :tada: 

I faced a small issue:
HPA fails due to deployment: `sentry-ingest-consumer` not found.
The deployment names are different for events, transactions and attachments.

Created separate HPAs for them leads to:
_pods by selector app=sentry,release=sentry,role=ingest-consumer are controlled by multiple HPAs_
Therefore changed the role name as well

S3 seems to be working fine for me so thought to remove the commented line saying that _s3 is unconfigured_

Also the deployment breaks as the fields are immutable. Therefore for me I changed the deployment names in my personal deployment but in this PR.
we can then bump the Chart version to a minor update. 
Any thoughts ?

kartik-nighania committed:
changed role names and separate HPAs added

kartik-nighania committed:
minor refactoring

kartik-nighania committed:
Update values.yaml

rajivmucheli commented:
Hey, are you running on the latest version with [all the services](https://github.com/sentry-kubernetes/charts/tree/develop/sentry/templates) in this repo ? i am having issues processing events upgrading from 9.1.2.

I raised a [request](https://github.com/sentry-kubernetes/charts/issues/1020) here but had no response

kartik-nighania commented:
> Hey, are you running on the latest version with [all the services](https://github.com/sentry-kubernetes/charts/tree/develop/sentry/templates) in this repo ? i am having issues processing events upgrading from 9.1.2.
> 
Yes I was able to run it with all the deployments and services. Did some modifications like:
* Used S3 for persistent storage between web and worker
* Used storage classes for DB deployment with Retain option
* Used my own Ingress of ALB controller
* Fixed some HPA issues that I found later
* Added IAM permissions by IAM roles for service accounts and modifying the existing SA

> I raised a [request](https://github.com/sentry-kubernetes/charts/issues/1020) here but had no response 
>
I looked at the request you raised but didn't see any error log mentioned. What is the issue you are getting ? 
Feel free to connect on a quick call on: [Linkedin Profile](https://www.linkedin.com/in/kartik-nighania-765227145/)

NikolaiIuzhaninYS commented:
These changes already present in new release https://github.com/sentry-kubernetes/charts/releases/tag/sentry-v23.0.0
This one might be closed

Mokto commented:
Thanks.
